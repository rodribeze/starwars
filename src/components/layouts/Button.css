.Button{
    background-color: white;
    padding: 10px;
    font-size: 1.2rem;
    text-decoration: none;
    border-radius: 3px;
}