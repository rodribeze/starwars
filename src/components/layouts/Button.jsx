import React from 'react'

import { Link } from 'react-router-dom';
import './Button.css'
const Button = props => {
    return (
        <Link className="Button" {...props} />
    )
};

export default Button;