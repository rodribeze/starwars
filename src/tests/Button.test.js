import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../components/layouts/Button'
import { BrowserRouter as Router } from 'react-router-dom'

test('Button is render', () => {

  const component = renderer.create(
    <Router>
    <Button to="/films/1">Ver mais filmes</Button>
    </Router>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});