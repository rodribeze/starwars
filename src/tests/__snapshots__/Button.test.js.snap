// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button is render 1`] = `
<a
  className="Button"
  href="/films/1"
  onClick={[Function]}
>
  Ver mais filmes
</a>
`;
